cmd/guaccollect/cmd/deps_dev.go

```diff
@@ -61,13 +67,20 @@ var depsDevCmd = &cobra.Command{
 			viper.GetBool("use-csub"),
 			viper.GetBool("service-poll"),
 			viper.GetBool("retrieve-dependencies"),
-			args)
+			args,
+			viper.GetBool("enable-prometheus"),
+			viper.GetString("prometheus-address"),
+		)
+		//print out all the flags
+		logger.Infof("viewing flags: %+v", opts)
+		logger.Infof("prometheus address: %s %s", opts.promeAddr, viper.GetString("prometheus-addr"))
```

Comment:
can we distinguish what the two strings are here to be more helpful to the user.

Reply:
👍